pages/faq.tsx

```diff
@@ -105,6 +105,17 @@ function FAQ({}: Props) {
             guide.
           </>
         </FAQEntry>
+        <FAQEntry title='I received a "Ferdium is damaged and can’t be opened. You should move it to the Trash” error message when trying to install Ferdium. What should I do?'>
+          <>
+            This error message is shown because the releases aren't yet signed. We are still working on getting an Apple developer license.
+            <br />
+            Meanwhile, you could check out{" "}
+            <ExternalLink href='https://github.com/ferdium/ferdium-app/issues/34'>
```

Comment:
There's an entry in the main `README.md` here for this https://github.com/ferdium/ferdium-app#unsigned-nightlies-on-macos - I prefer pointing to the README as opposed to an issue for such FAQ

Reply:
Thank you and it's a great idea, I didn't notice there is a section about it in the main `README.md`. I will reword this faq to be more general for both Windows and macOS, and link it to that section in the `README.md`, if that sounds good?

Reply:
yes - please go ahead! Thanks!